lib/main_common.pm

```diff
@@ -2304,6 +2306,7 @@ sub load_system_prepare_tests {
     # temporary adding test modules which applies hacks for missing parts in sle15
     loadtest 'console/sle15_workarounds' if is_sle('15+');
     loadtest 'console/integration_services' if is_hyperv || is_vmware;
+    loadtest 'console/check_os_release';
```

Comment:
hm, having this test module scheduled three times and also in cases where we actually do not need it, e.g. in the system image preparation jobs is something which I recommend to avoid. Can we just schedule this test module as part of extra_tests_in_textmode and according "extra tests" for sles4sap et al.? If you insist on having it as part of the not-extra tests then please crosscheck staging tests as well. I am not sure if the staging "product" has the expected data in /etc/os-release

Reply:
I agree with @okurz, just schedule it in console_extratests

Reply:
Wasn't sure where to schedule it to cover most cases. Figured that since all job groups have at least one `create_hdd_*` task, that this was the better way to make sure all products are covered, but I agree that this does not qualify as a system preparation task (or a x11 installation task for that matter). Will remove those 2 calls. Do you think I should also schedule it in `load_extra_tests_console` if it's already in `load_consoletests`?

Reply:
> I agree with @okurz, just schedule it in console_extratests

Same question as the one to @okurz: I feel this needs to be in `load_consoletests`. Should this also go in `load_extra_tests_console`?

Reply:
@alvarocarvajald Testing it twice won't make much sense :)... I guess either is ok... (I'm starting to look at `load_extra_tests_console` like an unwanted friend... due to it's size...

Reply:
It is true that `load_extra_tests_console` is a bit big however pretty simple. `load_consoletests` on the other hand is something I would consider "critical system integration" tests which should also block further execution of downstream tests and still checking `/etc/os-release` does not have that much *functional* impact. So I would go for *just* `load_extra_tests_console`

Reply:
OK, scheduling only in `load_extra_tests_console()` and in `load_sles4sap_tests()`, however I'm worried that this is leaving products (RT and HPC) without explicit checks. Hopefully both are being covered with `load_extra_tests_console()`.

Reply:
yeah, that's the thing: IMHO all relevant products should run *some* extra tests